interface-definitions/include/firewall/rule-log-options.xml.i

```diff
@@ -0,0 +1,88 @@
+<!-- include start from firewall/rule-log-options.xml.i -->
+<node name="log-options">
+   <properties>
+    <help>Log options</help>
+  </properties>
+  <children>
+    <leafNode name="group">
+      <properties>
+        <help>Set log group. Log must be enable.</help>
+        <valueHelp>
+          <format>u32:0-65535</format>
+          <description>Log group to send messages to</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 0-65535"/>
+        </constraint>
+      </properties>
+    </leafNode>
+    <leafNode name="snaplen">
+      <properties>
+        <help>Length of packet payload to include in netlink message. Log must be enable.</help>
+        <valueHelp>
+          <format>u32:0-4294967295</format>
```

Comment:
Having a packaged with length 4294967295 logged might cause a lot of trouble. I think we should limit this to 9000.